v03_pipeline/bin/pipeline_worker.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+import datetime
+import os
+import time
+
+import luigi
+
+from v03_pipeline.api.model import LoadingPipelineRequest
+from v03_pipeline.lib.logger import get_logger
+from v03_pipeline.lib.paths import (
+    loading_pipeline_queue_path,
+    project_pedigree_path,
+    project_remap_path,
+)
+from v03_pipeline.lib.tasks import (
+    UpdateCachedReferenceDatasetQueries,
+    UpdateVariantAnnotationsTableWithNewSamplesTask,
+    WriteProjectFamilyTablesTask,
+)
+
+logger = get_logger(__name__)
+
+
+def main():
+    while True:
+        try:
+            if not os.path.exists(loading_pipeline_queue_path()):
+                continue
+            with open(loading_pipeline_queue_path()) as f:
+                lpr = LoadingPipelineRequest.model_validate_json(f.read())
+            project_remap_paths = [
+                project_remap_path(
+                    lpr.reference_genome,
+                    lpr.dataset_type,
+                    lpr.sample_type,
+                    project_guid,
+                )
+                for project_guid in lpr.projects_to_run
+            ]
+            project_pedigree_paths = [
+                project_pedigree_path(
+                    lpr.reference_genome,
+                    lpr.dataset_type,
+                    lpr.sample_type,
+                    project_guid,
+                )
+                for project_guid in lpr.projects_to_run
+            ]
+            task_kwargs = {
+                k: v for k, v in lpr.model_dump().items() if k != 'projects_to_run'
+            }
+            tasks = [
+                UpdateCachedReferenceDatasetQueries(
```

Comment:
These are the three tasks necessary to get everything to run correctly.  They do not have any dependency structure between them, though they share upstream dependencies (like `UpdatedReferenceDataset` or `WriteImprotedCallset`).  

The blockers to condensing this to a single task are mostly around how we have split tasks apart in airflow.  We have a `for project_guid in projects_to_run` loop there so that each project task can be viewed in isolation.